accord-core/src/main/java/accord/coordinate/Execute.java

```diff
@@ -70,6 +71,12 @@ private void start()
     {
         Set<Id> readSet = readTracker.computeMinimalReadSetAndMarkInflight();
         Commit.commitAndRead(node, topologies, txnId, txn, homeKey, executeAt, deps, readSet, this);
+        // skip straight to persistence if there's no read
+        if (txn.read().keys().isEmpty())
```

Comment:
We are not good with nullability in Accord, so its not clear if read is allowed to be null... Based off Caleb's branch org.apache.cassandra.cql3.statements.TransactionStatement#createRead won't be null in a write only txn, but is null allowed or not? If so then we need a null check at "if (txn.read().keys().isEmpty())"